feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radius_proxy.c

```diff
@@ -0,0 +1,231 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#include <stdio.h>
+#include <stdbool.h>
+#include <time.h>
+
+#include <uci.h>
+#include <uci_blob.h>
+
+#include <target.h>
+
+#include <curl/curl.h>
+
+#include "ovsdb.h"
+#include "ovsdb_update.h"
+#include "ovsdb_sync.h"
+#include "ovsdb_table.h"
+#include "ovsdb_cache.h"
+
+#include "nl80211.h"
+#include "radio.h"
+#include "vif.h"
+#include "phy.h"
+#include "log.h"
+#include "evsched.h"
+#include "uci.h"
+#include "utils.h"
+#include "radius_proxy.h"
+
+ovsdb_table_t table_Radius_Proxy_Config;
+struct blob_buf uci_buf = {};
+struct blob_attr *n;
+
+enum {
+	RADIUS_PROX_CONFIG_NAME,
```

Comment:
Why is the prefix different for the first two in this enum? Consistency is good for readability and maintenance.